### DISCUSSION TITLE
Student life

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#376] [General]
How did you spend your student years?
I keep hearing that students have lots of fun and make parties but honestly don't know where to find time for all that. We are so busy with the studies and constant exams that I barely sleep.

**Comment by hernessteve** [comment]
For me it was 50/50. I had to work hard but i also always found time to have fun.
If you do only first one you will burn out very quickly.
The best tip i was ever given - find shortcuts  where it is possible. Here is a great service [https://www.topwritersreview.com/reviews/myassignmenthelp/](https://www.topwritersreview.com/reviews/myassignmenthelp/) which helped me with my essay assignments. Saved me a lot of time and nerves.

**Comment by FloatingBook** [comment]
A student life is a lifetime memory which only comes once, here we are exposed into the wide world of knowledge. I have saved a lot of time for other curricular activities and enjoy weekends by getting help from https://topgrademakers.com/ to write my essay assignments.
